src/components/Status.js

```diff
@@ -0,0 +1,13 @@
+import { useDispatch } from 'react-redux';
+import { checkStatusAction } from '../redux/categories/categories';
+
+const StatusButton = () => {
+  const dispatch = useDispatch();
+
+  const click = () => dispatch(checkStatusAction());
+  return (
+    <button type="button" className="check-status" onClick={click}>Check Status</button>
+  );
+};
```

Comment:
**Thanks for the feedback @Whoistolu but it was stated that StatusButton must be rendered in categories component and it will return "Under construction"**

<img width="650" alt="Screen Shot 2022-03-30 at 00 28 48" src="https://user-images.githubusercontent.com/40362530/160718114-6c5de44b-830a-4634-ab3e-739a79c2057f.png">
<img width="671" alt="Screen Shot 2022-03-30 at 00 37 21" src="https://user-images.githubusercontent.com/40362530/160718232-0dd01ed1-1965-4bfb-8cf8-fc156d0c9c4b.png">

**The issue was that i didn't implement rendering "Under construction"**